bsp4j/src/main/xtend-gen/ch/epfl/scala/bsp4j/PythonBuildTarget.java

```diff
@@ -0,0 +1,75 @@
+package ch.epfl.scala.bsp4j;
```

Comment:
This looks like a generated file. Are you sure it should be committed?

Reply:
All other generated files are commited AFAIK (eg. https://github.com/build-server-protocol/build-server-protocol/blob/master/bsp4j/src/main/xtend-gen/ch/epfl/scala/bsp4j/JvmBuildTarget.java)

Reply:
I think so, all the other generated files are commited (eg. https://github.com/build-server-protocol/build-server-protocol/blob/master/bsp4j/src/main/xtend-gen/ch/epfl/scala/bsp4j/ScalaBuildTarget.java)